README.md

```diff
@@ -78,4 +78,4 @@ This website was forked from the BSD-licensed [djangoproject.com](https://github
 
 ## Copyright
 
-Copyright OpenSearch Contributors. 
\ No newline at end of file
+Copyright OpenSearch Contributors. 
```

Comment:
Is this some sort of weird linting?

Reply:
I believe so. It appears to have removed and re-add this line.